barretenberg/cpp/src/barretenberg/polynomials/polynomial_arithmetic.test.cpp

```diff
@@ -999,6 +999,58 @@ TYPED_TEST(PolynomialTests, compute_efficient_interpolation)
         EXPECT_EQ(src[i], poly[i]);
     }
 }
+// Test efficient Lagrange interpolation when interpolation points contain 0
+TYPED_TEST(PolynomialTests, compute_efficient_interpolation_domain_with_zero)
+{
+    using FF = TypeParam;
+    constexpr size_t n = 15;
+    FF src[n], poly[n], x[n];
```

Comment:
It's better to use std::array than c-style arrays

Reply:
Changed it